torch/distributed/utils.py

```diff
@@ -330,3 +330,9 @@ def _replace_by_prefix(
         new_key = new_prefix + key[len(old_prefix) :]
         state_dict[new_key] = state_dict[key]
         del state_dict[key]
+
+def _get_device_module(device_type: str):
```

Comment:
@awgu FSDP has code that is almost identical to this one.

Can we unify it all here?

Reply:
I might have missed it -- where in FSDP code do we convert from string (e.g. `"cuda"`) to the Python module (e.g. `torch.cuda`)?

The closest thing I can see is the function `_init_device_handle()` with:
https://github.com/pytorch/pytorch/blob/4baac20117a86f811aa9acf5f8d9d3415faff238/torch/distributed/fsdp/_init_utils.py#L348